README.md

```diff
@@ -82,5 +79,48 @@ To set up your environment on Mac, using [Vagrant](https://www.vagrantup.com) is
 
 	sudo pip install ledgerblue==0.1.15
 	```
-    
+
 [1] *Note that this is because Docker for Mac does not support USB connectivity due to [xhyve limitations](https://github.com/mist64/xhyve#what-is-bhyve)*
+
+## Deploying
+
+The build process is managed with [Make](https://www.gnu.org/software/make/).
+
+### Setting Up Custom CA for signing Ledger App
+
+To remove the prompts when installing, running and deleting the app from the Ledger device, you need to generate a Custom CA keypair.
```

Comment:
Should we write a `ca.sh` script to do this? Make life a bit faster?

Reply:
For sure. But atm it only needs to be done once, so not too painful. Plus, I get the sense some awk would be needed as the output from the genCAPair comes in two lines - 1st publick key, 2nd private key. I'll add a story to the backlog. Be nice to have it as part of the vagrant provisioning script...

Reply:
Now we're talking!! I'm happy for this to be merged, regain some significant time from my life!